jiant/proj/main/modeling/model_setup.py

```diff
@@ -38,7 +39,7 @@ def setup_jiant_model(
     """
     model_arch = ModelArchitectures.from_model_type(model_type)
     transformers_class_spec = TRANSFORMERS_CLASS_SPEC_DICT[model_arch]
-    tokenizer = model_setup.get_tokenizer(model_type=model_type, tokenizer_path=tokenizer_path)
+    tokenizer = transformers.AutoTokenizer.from_pretrained(hf_pretrained_model_name)
```

Comment:
- Is `tokenizer_path` still being used?
- In some rare cases, we use tokenizers that aren't the pretrained HF ones. One option is the use `.from_pretrained`, but ensure that we also handle paths, like HF's `.from_pretrained` functions do.
- (On that note, consider streamlining the model loading/exporting to skip exporting if we're using the basic case of fine-tuning from the standard pretrained models)

Reply:
- `tokenizer_path` is no longer being used. Thanks for pointing this out.
- This should already be supported but missing documentation. I'll add that in.
- I completely agree we need to do this...was planning to do that in a different PR since the refactoring was getting convoluted when including that change in this PR